### ISSUE TITLE
Broken packages: nothing provides lib:libconfuse>=2.7

### ISSUE THREAD

**Issue by probonopd** [haikuports/haikuports#3997]
```
/> uname -a
Haiku shredder 1 hrev53259+3 Jul 15 2019 23:37:0 x86_64 x86_64 Haiku
/> pkgman i avrdude
100% repochecksum-1 [65 bytes]
Validating checksum for Haiku...done.
100% repochecksum-1 [64 bytes]
Validating checksum for HaikuPorts...done.
Encountered problems:
problem 1: nothing provides lib:libconfuse>=2.7 needed by libftdi-1.4-7
  solution 1:
    - do not install "providing avrdude"
Please select a solution, skip the problem for now or quit.
select [1/s/q]: 
```

**Comment by diversys** [comment]
Fixed in 937e2d40d360069c2de5beeb552b95e5c5463238.

**Comment by probonopd** [comment]
Small change, big effect. It is working for me now. Thank you!
Amazing how it takes less than an hour for a fix to propagate from here to being available for everyone.
